src/clojurewerkz/elastisch/rest/document.clj

```diff
@@ -264,9 +264,10 @@
 
 (defn scroll-seq
   "Returns a lazy sequence of all documents for a given scroll query"
-  ([^Connection conn prev-resp {:keys [search_type] scroll-ttl :scroll
-                                :or {scroll "1m"} :as opts}]
```

Comment:
FYI you didn’t _have_ to move the default value down below; it would've worked if you changed `:or {scroll "1m"}` to `:or {scroll-ttl "1m"}`.